blivet/devicefactory.py

```diff
@@ -69,6 +70,9 @@ def is_supported_device_type(device_type):
         :returns: True if this device type is supported
         :rtype: bool
     """
+    if device_type == DEVICE_TYPE_LVM_VDO:
+        return not any(e for e in LVMVDOPoolMixin._external_dependencies if not e.available)
```

Comment:
Why not just set `devices` to `LVMVDOPoolMixin` (and maybe also `LVMLogicalVolumeDevice`) below?

Reply:
I've tried to do that and didn't manage to make it work. With the `Mixin` classes not being subclasses of `Device` and `LVMLogicalVolumeDevice` inheriting from all the `Mixin` classes, it made Python really unhappy. I'm not saying it's impossible, but I gave up and did this instead :-)